src/scripts/cronjobs/emailBreachAlerts.tsx

```diff
@@ -393,19 +393,6 @@ async function pullMessages() {
 
   return [subClient, response.receivedMessages] as const;
 }
```

Comment:
Just to understand here, transitioning from task back to cron means that SRE will need to switch back to running `cron:breach-alerts`